packages/features/bookings/components/TeamFilter.tsx

```diff
@@ -0,0 +1,75 @@
+import { useLocale } from "@calcom/lib/hooks/useLocale";
```

Comment:
This file is 99% similar to the previous filter. Do you agree that it makes sense to abstract this into a reusable component? Makes adding more filters super easy 🔥 
Let me know if there's any difference between them I overlooked!

Reply:
#5590  created an issue along with other things that need to be picked up in the future